release-notes/9.0/linux-packages.json

```diff
@@ -0,0 +1,408 @@
+                    "command-parts": ["install", "-y", "{packageName}"]
+                }
+            ],
+            "releases": [
+                {
+                    "name": "Debian 12 (Bookworm)",
+                    "release": "12",
+                    "packages": [
+                        {
+                            "id": "libc",
+                            "name": "libc6"
+                        },
+                        {
+                            "id": "libgcc",
+                            "name": "libgcc-s1"
+                        },
+                        {
+                            "id": "ca-certificates",
+                            "name": "ca-certificates"
+                        },
+                        {
+                            "id": "libssl",
+                            "name": "libssl3"
+                        },
+                        {
+                            "id": "libstdc++",
+                            "name": "libstdc++6"
+                        },
+                        {
+                            "id": "libicu",
+                            "name": "libicu72"
+                        },
+                        {
+                            "id": "tzdata",
+                            "name": "tzdata"
+                        },
+                        {
+                            "id": "krb5",
+                            "name": "libgssapi-krb5-2"
+                        }
+                    ]
+                },
+                {
+                    "name": "Debian 11 (Bullseye)",
+                    "release": "11",
+                    "packages": [
+                        {
+                            "id": "libc",
+                            "name": "libc6"
+                        },
+                        {
+                            "id": "libgcc",
+                            "name": "libgcc1"
+                        },
+                        {
+                            "id": "ca-certificates",
+                            "name": "ca-certificates"
+                        },
+                        {
+                            "id": "libssl",
+                            "name": "libssl1.1"
+                        },
+                        {
+                            "id": "libstdc++",
+                            "name": "libstdc++6"
+                        },
+                        {
+                            "id": "libicu",
+                            "name": "libicu67"
+                        },
+                        {
+                            "id": "tzdata",
+                            "name": "tzdata"
+                        },
+                        {
+                            "id": "krb5",
+                            "name": "libgssapi-krb5-2"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "Ubuntu",
```

Comment:
I think it would be ideal to include Red Hat for the launch as well.

Reply:
Good call. Can you contribute those @omajid?

After we merge this PR, I'll add .NET 6/8 versions, too.

Reply:
Also, we want as many distros as possible. I just started with some handy ones to prove out the idea.